nexus/tests/integration_tests/switch_port.rs

```diff
@@ -24,6 +24,10 @@ use omicron_common::api::external::{
 type ControlPlaneTestContext =
     nexus_test_utils::ControlPlaneTestContext<omicron_nexus::Server>;
 
+// TODO: unfortunately this test can no longer be run in the integration test
+//       suite because it depends on communicating with MGS which is not part
+//       of the infrastructure available in the integration test context.
+#[ignore]
```

Comment:
Are we unable to use the faux MGS here (the same one we use to simulate MGS for the switch zone with SoftNPU)?

Reply:
I'm sure that can be made to work, but it's not a part of the current integration test infrastructure and not something I have the bandwidth to take on at the moment.

Reply:
Totally understandable. It may be something that fits into the scope of work for the next set of RPW PRs, I was curious if there were any blockers I wasn't aware of.

Reply:
For what it's worth, as of (I think it was) #4291, the test suite starts a copy of MGS backed by simulated SPs.  (I haven't looked at what this test requires so I'm not sure that will work but it may be an option.)

Reply:
Cool. I'll give this a shot when I merge main back into this branch.